lua/wikis/commons/Transfer/References.lua

```diff
@@ -310,10 +313,12 @@ function TransferRef._getTextAndLink(reference, options)
 	elseif refType == INSIDE_TYPE then
 			return 'Liquipedia has gained this information from a trusted inside source.'
 	elseif refType == CONTRACT_TYPE then
-		link = 'https://docs.google.com/spreadsheets/d/1Y7k5kQ2AegbuyiGwEPsa62e883FYVtHqr6UVut9RC4o/pubhtml#'
-		local appendedText = 'LoL Esports League-Recognized Contract Database'
+		local contractDatabaseRef = (Info.config.transfers or {}).contractDatabaseRef
+		assert(contractDatabaseRef, 'Contract database type is not available')
```

Comment:
this will error on wikis that do not have that set in Module:Info
are you sire there do not exist such cases?